model-desc/cds-model-props.yml

```diff
@@ -10905,11 +10904,14 @@ PropDefinitions:
       Value: Program Short Name Text
       Version: '1.00'
   # proteomic
-  aliquot_id:
-    Desc: One or more characters used to identify a portion of a sample.
+  protemic_info_id:
```

Comment:
I think this should be "proteomic_info_id"

Reply:
this is fixed, thanks